Topic4/src/classes/WebUserService.java

```diff
@@ -0,0 +1,117 @@
+package classes;
+
+import java.awt.Image;
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.List;
+import interfaces.UserService;
+
+public class WebUserService implements UserService {
+
+	private List<WebUser> users;
+
+	public WebUserService() {
+		users = new ArrayList<WebUser>();
+	}
+
+	@Override
+	public WebUserServiceResult createUser(String mail, String password, String name, String lastName) {
+		WebUser user = readUser(mail);
+		WebUserServiceResult result;

```

Comment:
Another option could be to set in this line the fail WebUserServiceResult to avoid the need to have an else below (it's only a comment, you don't need to change it)
WebUserServiceResult result = new WebUserServiceResult(1, "Fail, existent user");